File: algoritmos-e-logica-de-programacao/PyCharm/ws-pycharm/saida-de-dados.py
# x = "Maria"
# y = 19

# x: int; y: int

# x = 10
# y = 20

# x: float

# x = 2.3456

idade: int
salario: float
nome: str
sexo: str

idade = 32
salario = 4560.9
nome = "Maria Silva"
sexo = "F"

# print("Bom dia", end='') # Sem quebra de linha, colado
# print("Boa noite")

#print("Bom dia") # Com quebra de linha
# print("Boa noite")

# print("%s tem %d anos" % (x, y))

# print(x)
# print(y)

# print("{:.2f}".format(x))

print(f"A funcionaria {nome}, sexo {sexo}, ganha {salario:.2f} e tem {idade} anos") # Interpolação
print("A funcionaria {:s}, sexo {:s}, ganha {:.2f} e tem {:d} anos".format(nome, sexo, salario, idade)) # Placeholders

File: algoritmos-e-logica-de-programacao/Eclipse/ws-eclipse/curso_programacao/src/SaidaDeDados.java
import java.util.Locale;

public class SaidaDeDados {

public static void main(String[] args) {
		Locale.setDefault(Locale.US);
	
		// int x, y;
		
		// double x;
		
		int idade;
		double salario;
		String nome;
		char sexo;
		
		// System.out.println("Bom dia"); // Sem quebra de linha
		// System.out.println("Boa noite");
		
		// System.out.println("Bom dia"); // Com quebra de linha
		// System.out.println("Boa noite");

		// x = 10;
		// y = 20;
		
		// x = 2.3456;
		
		idade = 32;
		salario = 4560.9;
		nome = "Maria Silva";
		sexo = 'F';
		
		// System.out.println(x);
		// System.out.println(y);
		
		// System.out.println(String.format("%.2f", x));
		
		System.out.println("A funcionaria " + nome + ", sexo " + sexo + ", ganha " + String.format("%.2f", salario) + " e tem " + idade + " anos");
	}

}
